public/static/js/init.py

```diff
@@ -8,10 +8,13 @@
 import json
 import re
 from collections import deque
-from pyodide import to_js
+from pyodide.ffi import to_js
```

Comment:
random change related to Pyodide moving the to_js method around (this removed the deprecation warning when the site loads)